src/etc/licenseck.py

```diff
@@ -57,18 +47,17 @@
 
 def check_license(name, contents):
     # Whitelist check
-    for exception in exceptions:
-        if name.endswith(exception):
-            return True
+    if any(name.endswith(e) for e in exceptions):
+        return True
 
     # Xfail check
     firstlineish = contents[:100]
-    if firstlineish.find("ignore-license") != -1:
+    if "ignore-license" in firstlineish:
         return True
 
     # License check
     boilerplate = contents[:500]
```

Comment:
If the copyright lines *do* get removed, `500` can probably be decreased to `350`, but doing so in this PR causes a failing build since the copyright lines are currently still there.